src/styles/components/_home.scss

```diff
@@ -5,6 +5,10 @@ $breakpoint-tablet: 820px;
   grid-template-columns: 1fr;
   gap: 1rem;
 
+  ul {
```

Comment:
Rather than applying this to every `ul` within `.home`, I would be more targeted with where the style is applied. Otherwise, any other ul that could very likely be added within `.home` is going to get this class. There are different ways to choose from:
 1. Add a class to the ul, e.g. `masonry-grid`, and apply your mixin to that. It makes sense that this grid would have its own class, for purposes of re-use.
 2. If using a more utility class approach to styles, the mixin could be part of a class like `bg-pattern`, that is added to the `ul`
 3. If adding a class is not wanted for some reason, be more targeted with the selector like `.home > ul` that only targets the lists that are direct children (not totally future proof if another unrelated list gets added in)